### PR TITLE
Migrate old WebSockets data from MDN docs

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Continuation of https://github.com/mdn/browser-compat-data/pull/3607#discussion_r267880508.

### CODE REVIEW

**Comment by ddbeck** [comment]
OK, I dug into this a bit and this is my sense of what's going on here:

There were many revisions to the WebSocket protocol. Browsers implemented only a handful of them, but it's difficult to tell which browsers implemented which version because of inconsistent numbering and naming conventions. I found it very difficult to corroborate the migrated data about older versions of the protocol and I couldn't figure out whether any of the browsers _continued_ to support those older versions.

Which browsers supported which draft of the protocol was apparently a very interesting question circa 2011 but, thankfully, there is now a stable standard in the form of [RFC 6455](https://tools.ietf.org/html/rfc6455), which corresponds to the header version number 13. Tracking down support for v13 was relatively straightforward and I was able to substantiate or refine that data.

Given all this, I'm going to review with suggestions such that we only cover the RFC 6455 version and call it a day. 

<details>
<summary>Details on RFC 6455 support</summary>

  * Chrome 16: https://www.chromestatus.com/features/6555138000945152
  * Firefox 11: https://bugzilla.mozilla.org/show_bug.cgi?id=666349
  * Opera 12.50 (hoist to 15): https://web.archive.org/web/20120805234006/http://my.opera.com/ODIN/blog/2012/08/03/a-hot-opera-12-50-summer-time-snapshot
  * Safari 6: I couldn't find anything authoritative on the version number but this bug https://bugs.webkit.org/show_bug.cgi?id=69910 seems consistent with the version 6 release date (for both macOS and iOS)
  * IE 10: https://blogs.msdn.microsoft.com/ie/2012/03/19/websockets-in-windows-consumer-preview/
</details>

File: api/WebSocket.json [hunk 1]
```diff
@@ -726,6 +726,246 @@
             "deprecated": false
           }
         }
+      },
+      "version_76_support": {
```
Comment (ddbeck) [inline api/WebSocket.json]: Drop this feature

File: api/WebSocket.json [hunk 2]
```diff
@@ -726,6 +726,246 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "protocol_v7_support": {
```
Comment (ddbeck) [inline api/WebSocket.json]: Drop this feature

File: api/WebSocket.json [hunk 3]
```diff
@@ -726,6 +726,246 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "protocol_v10_support": {
```
Comment (ddbeck) [inline api/WebSocket.json]: Drop this feature

File: api/WebSocket.json [hunk 4]
```diff
@@ -726,6 +726,246 @@
...
+            "deprecated": true
+          }
+        }
+      },
+      "rfc_6455_support": {
+        "__compat": {
+          "description": "Standard - RFC 6455 Support",
+          "support": {
+            "chrome": {
+              "version_added": "43"
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": "16"
```

File: api/WebSocket.json [hunk 5]
```diff
@@ -726,6 +726,246 @@
...
+            },
+            "chrome_android": {
+              "version_added": "43"
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": "18"
```

File: api/WebSocket.json [hunk 6]
```diff
@@ -726,6 +726,246 @@
...
+            },
+            "edge": {
+              "version_added": "14"
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": "12"
```

File: api/WebSocket.json [hunk 7]
```diff
@@ -726,6 +726,246 @@
...
+            },
+            "edge_mobile": {
+              "version_added": false
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": "12"
```

File: api/WebSocket.json [hunk 8]
```diff
@@ -726,6 +726,246 @@
...
+            },
+            "firefox": {
+              "version_added": "48"
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": "11"
```

File: api/WebSocket.json [hunk 9]
```diff
@@ -726,6 +726,246 @@
...
+            },
+            "firefox_android": {
+              "version_added": "48"
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": "11"
```

File: api/WebSocket.json [hunk 10]
```diff
@@ -726,6 +726,246 @@
...
+            },
+            "ie": {
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "12.1"
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": "15"
```

File: api/WebSocket.json [hunk 11]
```diff
@@ -726,6 +726,246 @@
...
+            },
+            "opera_android": {
+              "version_added": "12.1"
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": "15"
```

File: api/WebSocket.json [hunk 12]
```diff
@@ -726,6 +726,246 @@
...
+            },
+            "safari": {
+              "version_added": "6"
+            },
+            "safari_ios": {
+              "version_added": "6"
+            },
+            "webview_android": {
+              "version_added": "43"
```
Comment (ddbeck) [inline api/WebSocket.json]:
```suggestion
              "version_added": true
```

**Comment by vinyldarkscratch** [comment]
Of course, and thanks for looking into this more!  I've applied all of your suggestions and dropped all the features that refer to older versions.  :wink:

**Comment by vinyldarkscratch** [comment]
I'm not certain why the Travis build is failing, as it's working on my end...

File: api/WebSocket.json (hunk 6 above)
Comment (vinyldarkscratch) [reply api/WebSocket.json]: Turns out that there's no 11, 12, or 13 for Firefox Android.

**Pull request merged**